src/mpid/ch4/netmod/ucx/ucx_init.c

```diff
@@ -229,7 +229,8 @@ int MPIDI_UCX_mpi_init_hook(int rank, int size, int appnum, int *tag_bits, MPIR_
 
     ucp_params.field_mask = UCP_PARAM_FIELD_FEATURES |
         UCP_PARAM_FIELD_REQUEST_SIZE |
-        UCP_PARAM_FIELD_ESTIMATED_NUM_EPS | UCP_PARAM_FIELD_REQUEST_INIT;
+        UCP_PARAM_FIELD_ESTIMATED_NUM_EPS |
+        UCP_PARAM_FIELD_REQUEST_INIT | UCP_PARAM_FIELD_MT_WORKERS_SHARED;
```

Comment:
What implications does that have to overheads in the UCX library? Should `UCP_PARAM_FIELD_MT_WORKERS_SHARED` be conditionally added based on MPICH configuration?

Reply:
Good point, that can be done.

Reply:
fixed.